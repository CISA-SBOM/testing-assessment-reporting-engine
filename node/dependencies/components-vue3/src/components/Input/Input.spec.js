import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from './Input.vue'

describe('Input', () => {
  it('should match its default snapshot', () => {
    const wrapper = shallowMount(Component, {})
    expect(wrapper.element).toMatchSnapshot()
  })
})
