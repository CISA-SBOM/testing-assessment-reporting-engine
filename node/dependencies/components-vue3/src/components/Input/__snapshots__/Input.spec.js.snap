// Vitest Snapshot v1

exports[`Input > should match its default snapshot 1`] = `
<div
  data-id="sds-input"
>
  <input
    class="form-control"
    maxlength="524288"
    placeholder=""
    type="text"
  />
  <!---->
</div>
`;
